spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'takeaway'
+
+describe 'User Stories' do
+  let(:takeaway) { Takeaway.new(phone, menu, order) }
+  let(:phone) { Dotenv.load['TWILIO_PHONE'] }
+  let(:menu) { Menu.new }
+  let(:order) { Order.new }
+
+  # As a customer
+  # So that I can check if I want to order something
+  # I would like to see a list of dishes with prices
+  it 'displays a list of dishes with prices' do
+    expect(takeaway.read_menu).to eq menu.display_menu
+  end
+
+  # As a customer
+  # So that I can order the meal I want
+  # I would like to be able to select some number of several available dishes
+  it 'can select some number of several available dishes' do
+    dishes = {chicken_tikka_masala: 1, naan: 2}
+    expect{ takeaway.select_dishes(dishes) }.not_to raise_error
+  end
+
+  it 'cannot select unavailable dish' do
+    expect(takeaway.select_dishes(bouillabaisse: 5)).to be_empty
+  end
+
+  # As a customer
+  # So that I can verify that my order is correct
+  # I would like to check that the total I have been given matches the sum of the various dishes in my order
```

Comment:
Metrics/LineLength: Line is too long. [108/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)